src/topology/continuous_function/algebra.lean

```diff
@@ -10,11 +10,13 @@ import algebra.algebra.subalgebra
 /-!
 # Algebraic structures over continuous functions
 
-In this file we define instances of algebraic structures over continuous functions. Instances are
-present both in the case of the subtype of continuous functions and the type of continuous bundled
-functions. Both implementations have advantages and disadvantages, but many experienced people in
-Zulip have expressed a preference towards continuous bundled maps, so when there is no particular
-reason to use the subtype, continuous bundled functions should be used for the sake of uniformity.
+In this file we define instances of algebraic structures over the type `continuous_map α β`
+(denoted `C(α, β)`) of **bundled** continuous maps from `α` to `β`. Note that we don't provide
+instances about `{ f : α → β | continuous f }` anymore, because `C(α, β)` should be used instead.
+
+However, we also define subobject of `α → β` with carrier `{ f : α → β | continuous f }` because
+we need a subset of `α → β` to define a subobject of it (see for example `continuous_subring`,
+`continuous_subalgebra`, ...).
```

Comment:
What does this last comment "we also define a subobject of `α → β` etc" mean?  I don't think you ever define 
```lean
def continuous_map_subtype := { f : α → β | continuous f }
```
(and nor should you!  but that's how I would interpret that comment).

Reply:
I meant `subgroup`, `subring`, etc... But indeed it is ambiguous, do you have a better suggestion ?

Reply:
It's hard to phrase but here's my attempt:

> In this file we define instances of algebraic structures over the type `continuous_map α β` (denoted `C(α, β)`) of **bundled** continuous maps from `α` to `β`.  For example, `C(α, β)` is a group when `β` is a group, a ring when `β` is a ring, etc.
>
> For each type of algebraic structure, we also define an appropriate subobject of `α → β` with carrier `{ f : α → β | continuous f }`.  For example, when `β` is a group, a subgroup `continuous_subgroup` of `α → β` is constructed with carrier `{ f : α → β | continuous f }`.
>
>Note that, rather than using the derived algebraic structures on these subobjects (for example, when `β` is a group, the derived `group` structure on `continuous_subgroup`), one should use `C(α, β)` with the appropriate instance of the structure.

Reply:
That looks better, thanks !